### DISCUSSION TITLE
On Mac and I don't have permission?

### DISCUSSION THREAD

**Discussion by leonatkins** [ORelio/Minecraft-Console-Client#2176] [General]
When I do `./MinecraftClient` it says "zsh: permission denied: ./MinecraftClient" what do I do?

**Comment by ReinforceZwei** [comment]
`sudo chmod a+x ./MinecraftClient`

**Reply by leonatkins** [reply]
It says "Password:"

**Reply by ReinforceZwei** [reply]
Enter your Mac's password

**Reply by leonatkins** [reply]
> Enter your Mac's password

I did it still says permission denied.
